src/fsharp/FSharp.Core/seq.fsi

```diff
@@ -233,8 +335,15 @@ namespace Microsoft.FSharp.Collections
         /// is requested.</remarks>
         ///
         /// <param name="generator">The generating function for the sequence.</param>
+        /// <returns>The result sequence.</returns>
+        ///
+        /// <example>
+        /// <code lang="fsharp">
+        ///     [(fun () -> Seq.ofList [1; 2; 3]) |> Seq.delay // evaluates to seq [1; 2; 3], calling the gnerator every time.
```

Comment:
Extra leading `[`

You wouldn't use pipelining with `Seq.delay`

Reply:
I'll change, but would like to know why, if you don't mind.